chat-commands.js

```diff
@@ -748,6 +748,15 @@ exports.commands = {
 			}
 		}
 
+		if (targetRoom.subRooms && targetRoom.subRooms.size) {
+			for (room of targetRoom.subRooms) {
+				room.parent = null;
```

Comment:
This can still be a one-liner, you know.

     for (const subRoom of targetRoom.subRooms) subRoom.parent = null;

Also you forgot the `const`/`let`, which made this overwrite `room`, which is probably undesirable.

Reply:
oh shit yeah my bad